paxml/tasks/lm/params/nvidia.py

```diff
@@ -31,6 +31,111 @@
 
 WeightInit = base_layer.WeightInit
 
+@experiment_registry.register
+class NVIDIA1_3B(c4.TransformerLmSpmdAdam, lm_cloud.SyntheticDataset):
+  """Pipelined Transformer using Adam optimizer."""
+  USE_FLASH_ATTENTION = False
+  USE_TRITON_LAYER_NORM = False
+
+  USE_REPEATED_LAYER = True
+
+  DCN_MESH_SHAPE = [1, 1, 1]
+  ICI_MESH_SHAPE = [16, 1, 1]
+
+  PERCORE_BATCH_SIZE = 4
+
+  NUM_LAYERS = 24
+  NUM_HEADS = 32
+  DIMS_PER_HEAD = 64
+  MODEL_DIMS = 2048
+  HIDDEN_DIMS = 8192
+
+  MAX_SEQ_LEN = 2048
+  VOCAB_SIZE = 51200
+  ACTIVATION_CLS = layers.GELU
+  USE_GATED_ACTIVATION = False
+  PACKED_INPUT = True
+
+  FPROP_DTYPE = jnp.bfloat16
+
+  TRAINABLE_POSITION_EMB = True
+  TRAINABLE_PE_MAX_SEQ_LEN = MAX_SEQ_LEN
+
+  INIT_STD = 0.023
+  SOFTMAX_INIT_STD = 0.023
+
+  # optimizer related
+  LEARNING_RATE = 6e-4
+  ADAM_BETA1 = 0.9
+  ADAM_BETA2 = 0.95
+  ADAM_EPSILON = 1e-8
+  ADAM_EPSILON_ROOT = 0.0
+  CLIP_GRADIENT_NORM_TO_VALUE = 1.0
+  CLIP_THRESHOLD = 1.0
+
+  # Learning rate schedule
+  LR_SCHEDULE = 'linear_rampup_cosine_decay'
+  LR_COS_WARMUP = 0
+  LR_COS_DECAY_START = 1
+  LR_COS_DECAY_END = 500000
+  LR_COS_MIN_RATIO = 0.1
+  LR_COS_MAX = 1.0
+
+  def task(self) -> tasks_lib.SingleTask.HParams:
```

Comment:
I think we need to replace `tasks_lib.SingleTask.HParams` with `pax_fiddle.Config[tasks_lib.SingleTask]` now